.studiorc

```diff
@@ -127,3 +162,100 @@ function install_cookstyle() {
   RUBY_BIN="$(hab pkg path core/ruby)/bin"
   export PATH="$RUBY_BIN:$PATH"
 }
+
+document "pre_commit_verify" <<DOC
+  Runs all verify steps on local code before commiting or creating pr.
+DOC
+function pre_commit_verify() {
+  ( cd /src || return 1
+    log_line "Running Static Tests..."
+    static_errors=$(static_tests 2>&1 > /dev/null)
+    static_offenses=$?
+    
+    echo -en "\033[1A\033[14C\033[0KStatic Tests:\033[6C"
```

Comment:
Lots of nice colors, I would suggest using the standards, here are some examples:

https://github.com/chef/studio-common/blob/master/dot-studio/logging#L19
# ANSI Escape Codes:

Just so you can be pretty in your messages :smile:

### Example with `echo`
```
echo -e "I $(red love) $(blue chef) $(yellow friends)!"
```

### Or using the `log_line` helper:
```
log_line "I $(red love) $(green chef) $(yellow friends)!"
```

![image](https://user-images.githubusercontent.com/5712253/74886322-07f39f00-5335-11ea-8ea2-6d201c2a07cb.png)

Reply:
The escapes your seeing there are for re-writting the prior line to pass/fail output not color